src/components/LearnModal.jsx

```diff
@@ -0,0 +1,88 @@
+import { Button, Modal } from 'react-bootstrap';
+import { useState, useEffect } from 'react';
+import { instructions } from '../data/modal-instructions';
+
```

Comment:
Creating instructions in a different file and importing feels like such a clever way to do this! I really like this approach!